.github/workflows/build_pr_documentation.yaml

```diff
@@ -9,9 +9,10 @@ concurrency:
 
 jobs:
   build:
-    uses: huggingface/doc-builder/.github/workflows/build_pr_documentation.yml@main
+    uses: huggingface/doc-builder/.github/workflows/build_pr_documentation.yml@test-build-pr-outside-container
     with:
       commit_sha: ${{ github.event.pull_request.head.sha }}
       pr_number: ${{ github.event.number }}
       package: huggingface_hub
       languages: cn de fr en hi ko
+      container_image: ""
```

Comment:
when it is empty string, does it gets treated as a supplied value OR does it gets treated as default value (which is transformers container) ?

Reply:
This is what I am tested as it's not documented :smile:

Reply:
But it looks like it's treated as an empty value, so container is not used at all: https://github.com/huggingface/huggingface_hub/actions/runs/8423002045/job/23063619212?pr=2140 (i.e. it just runs on the base ubuntu:latest).